src/components/Roadmap/Roadmap.jsx

```diff
@@ -24,11 +25,22 @@ const Roadmap = () => {
   const [user1Pic, setUser1Pic] = useState(user1Picture); //replace user1 with null instead of image placeholder
   const [user2Pic, setUser2Pic] = useState(user2Picture); //replace user2 with null instead of image placeholder
 
+
   const [user1Name, setUser1Name] = useState("User1");
   const [user2Name, setUser2Name] = useState("User2");
   const [completionPercentage, setCompletionPercentage] = useState(0);
 
+useEffect(() => {
+  // Prompt confirmation when reload page is triggered
+  window.onbeforeunload = () => {
```

Comment:
Hm not sure if we need this. Unless I am wrong the user is not filling out some information that could be lost by exiting?

Reply:
Everytime my browser refreshed i'd have to fill out the form when I want to check if anything was affected when moving stuff around, so I tried to stop it; I'll remove it now

Reply:
Yeah I just don't know if it is a form. if it really is affecting it then add it in.